twentytwentyone-blocks/experimental-theme.json

```diff
@@ -0,0 +1,203 @@
+					},
+					{
+						"slug": "extra-large",
+						"size": "40px"
+					},
+					{
+						"slug": "huge",
+						"size": "96px"
+					},
+					{
+						"slug": "gigantic",
+						"size": "144px"
+					}
+				],
+				"spacing": {
+					"customPadding": true,
+					"units": [ "px", "em", "rem", "vh", "vw" ]
+				}
+			},
+			"custom": {
+				"font-primary": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+				"font-weight-light": 300,
+				"color": {
+					"primary": "var(--wp--preset--color--dark-gray)",
+					"secondary": "var(--wp--preset--color--gray)",
+					"primary-hover": "var(--wp--custom--color-primary)",
+					"background": "var(--wp--preset--color--green)",
+					"border": "var(--wp--custom--color--primary)"
+				},
+				"line-height": {
+					"body": 1.7,
+					"heading": 1.3,
+					"page-title": 1.1
+				},
+				"responsive": {
+					"aligndefault-width": "610px",
+					"alignwide-width": "1240px"
+				},
+				"spacing": {
+					"unit": "20px",
+					"horizontal": "25px",
+					"vertical": "30px"
+				}
+			}
+		},
+		"styles": {
+	    	"color": {
+				"background": "var(--wp--custom--color--background)",
+				"text": "var(--wp--custom--color--primary)",
+				"link": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--normal)",
+				"lineHeight": "var(--wp--custom--line-height--body)"
+			}
+		}
+	},
+	"core/paragraph": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)",
+				"link": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--normal)",
+				"lineHeight": "var(--wp--custom--line-height--body)"
+			}
+		}
+	},
+	"core/heading/h1": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--gigantic)",
+				"lineHeight": 1.1
+			}
+		}
+	},
+	"core/heading/h2": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--extra-large)",
+				"lineHeight": "var(--wp--custom--line-height--heading)"
+			}
+		}
+	},
+	"core/heading/h3": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "calc(1.25 * var(--wp--preset--font-size--large))",
+				"lineHeight": "var(--wp--custom--line-height--heading)"
+			}
+		}
+	},
+	"core/heading/h4": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--large)",
+				"lineHeight": "var(--wp--custom--line-height--heading)"
+			}
+		}
+	},
+	"core/heading/h5": {
+		"styles": {
+			"color": {
+				"text": "var(--wp--custom--color--primary)"
+			},
+			"typography": {
+				"fontSize": "var(--wp--preset--font-size--small)",
```

Comment:
It seems like writing styles in theme.json is much more verbose when compared to CSS, when we need to apply the same style to multiple selectors. Should we have a mechanism in theme.json to target multiple selectors? cc: @mtias, @nosolosw